### PR TITLE
Apply changes for updating OpenEmbedded support to r36

### PR DESCRIPTION
**Pull request by andre-rosa**

### CODE REVIEW

**Comment by andre-rosa** [comment]
```
Name                                               Stmts   Miss  Cover   Missing
--------------------------------------------------------------------------------
superflore/CacheManager.py                            20      0   100%
superflore/PackageMetadata.py                         25      0   100%
superflore/TempfileManager.py                         33      6    82%   49-57
superflore/__init__.py                                10      4    60%   5-8
superflore/docker.py                                  78     57    27%   28-31, 34-36, 39, 42, 45-51, 56-69, 72, 75, 78-88, 91-129, 134
superflore/exceptions.py                              15      2    87%   28, 33
superflore/generate_installers.py                     62      2    97%   45-46
superflore/generators/__init__.py                      0      0   100%
superflore/generators/bitbake/__init__.py              3      1    67%   3
superflore/generators/bitbake/gen_packages.py        111     90    19%   39-118, 125-180, 187-195, 201
superflore/generators/bitbake/oe_query.py            104     12    88%   58, 70, 121, 140-150, 154
superflore/generators/bitbake/ros_meta.py             32     23    28%   23-27, 30-36, 39-49, 52, 55, 58-60
superflore/generators/bitbake/run.py                 139    114    18%   43-232
superflore/generators/bitbake/yocto_recipe.py        537    467    13%   68-125, 128-132, 135-147, 150-155, 158-160, 163-169, 172-178, 181-187, 190-196, 199-205, 208-214, 222-225, 230-232, 235-237, 240, 243-245, 249, 253, 261-277, 282-289, 294-302, 306-363, 370-500, 504-505, 510-533, 538-655, 659-702, 706-722, 726-739, 743-792, 796-804
superflore/generators/ebuild/__init__.py               3      1    67%   3
superflore/generators/ebuild/ebuild.py               185     19    90%   96-99, 126-131, 146, 176-180, 186, 193, 206-210
superflore/generators/ebuild/gen_packages.py         141    113    20%   47-120, 126-138, 144-189, 194-211, 214, 217
superflore/generators/ebuild/metadata_xml.py          33      0   100%
superflore/generators/ebuild/overlay_instance.py      46     34    26%   29-34, 37-47, 52-75, 78-80
superflore/generators/ebuild/run.py                  119     97    18%   40-188
superflore/parser.py                                  16      0   100%
superflore/repo_instance.py                           72     52    28%   32-54, 61-70, 73-81, 87-88, 94, 100-101, 107, 110-129, 132
superflore/rosdep_support.py                          37      3    92%   44, 88-89
superflore/test_integration/__init__.py                0      0   100%
superflore/test_integration/gentoo/__init__.py         3      1    67%   3
superflore/test_integration/gentoo/build_base.py      28     19    32%   26-28, 33, 38-54
superflore/test_integration/gentoo/main.py            32     26    19%   24-80
superflore/utils.py                                  200     30    85%   59-62, 66-80, 84-96, 111, 221-222, 226, 278, 283, 300-301, 306
--------------------------------------------------------------------------------
TOTAL                                               2084   1173    44%
```

**Comment by andre-rosa** [comment]
Fixes #204

File: superflore/generators/bitbake/run.py [hunk 1]
```diff
@@ -91,6 +91,7 @@ def main():
         overlay = RosMeta(
             _repo,
             not args.output_repository_path,
+            branch='superflore/{}'.format(now),
```
Comment (herb-kuta-lge) [inline superflore/generators/bitbake/run.py]: It was intended that this <DATETIME> be identical to the one which is assigned to `ROS_SUPERFLORE_GENERATION_DATETIME`.

Comment (andre-rosa) [reply superflore/generators/bitbake/run.py]: Yes, thank you!

**Comment by andre-rosa** [comment]
> @andre-rosa Please reduce this diff to only fix the issue. There's a lot of changes in `utils.py`, for example, that seem to have little to do with supporting open embedded r36...

I've split a few changes into #206, #208, #210, #212, #214, #216, #218. Please take a look. Once we go through these, I'll rebase the remaining changes accordingly.

**Comment by herb-kuta-lge** [comment]
> Once we go through these, I'll rebase the remaining changes accordingly.

@andre-rosa < All these PRs have been merged. Are you able to rebase the remainder of the commits for this PR, or do you want me to do it?

**Comment by andre-rosa** [comment]
> > Once we go through these, I'll rebase the remaining changes accordingly.
> 
> @andre-rosa < All these PRs have been merged. Are you able to rebase the remainder of the commits for this PR, or do you want me to do it?

Thank you, @herb-kuta-lge ! I had to create another review because github didn't allow me to change the origin branch, but only the destination. Please take a look at #222 , thanks.

**Comment by andre-rosa** [comment]
Superseded by #222 .

**Pull request closed without merging**